openapi/paths/bulkobjects@access@{access_id}.yaml

```diff
@@ -0,0 +1,40 @@
+post:
+  summary: Get URLs for fetching bytes from multiple objects
+  description: >-
+    Returns an array of URL objects that can be used to fetch the bytes of multiple `DrsObject`s.
+
+    This method only needs to be called when using an `AccessMethod` that contains an `access_id`
+    (e.g., for servers that use signed URLs for fetching object bytes).
+  operationId: GetBulkAccessURL
+  parameters:
+    - $ref: '../components/parameters/AccessId.yaml'
+  responses:
+    200:
+      $ref: '../components/responses/200OkAccesses.yaml'
+    202:
+      $ref: '../components/responses/202Accepted.yaml'
+    400:
+      $ref: '../components/responses/400BadRequest.yaml'
+    401:
+      $ref: '../components/responses/401Unauthorized.yaml'
+    403:
+      $ref: '../components/responses/403Forbidden.yaml'
+    404:
+      $ref: '../components/responses/404NotFoundAccess.yaml'
+    500:
+      $ref: '../components/responses/500InternalServerError.yaml'
+  tags:
+    - Objects
+  x-swagger-router-controller: ga4gh.drs.server
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            object_ids:
```

Comment:
How about renaming this property to `selection` to harmonize with the DRS + Passport downscoping thread. 

@kwrodarmer @mbarkley WDYT?

Reply:
So "object_ids" --> "selection"?  I'm fine with that...

Reply:
It started as selection and, with some discussion,  we consciously went to object_ids as there may be other use cases than "selection" for how you get to a list of ids that you want to request. object_ids is neutral as to use case, rather than projecting a particular use case on to DRS which has some generality.